pkg/controller/statefulset/stateful_set.go

```diff
@@ -419,7 +419,7 @@ func (ssc *StatefulSetController) sync(key string) error {
 func (ssc *StatefulSetController) syncStatefulSet(set *apps.StatefulSet, pods []*v1.Pod) error {
 	glog.V(2).Infof("Syncing StatefulSet %v/%v with %d pods", set.Namespace, set.Name, len(pods))
 	if err := ssc.control.UpdateStatefulSet(set, pods); err != nil {
-		glog.V(2).Infof("Error syncing StatefulSet %s/%s with %d pods : %s", set.Namespace, set.Name, err)
+		glog.V(2).Infof("Error syncing StatefulSet %s/%s with %d pods : %s", set.Namespace, set.Name, len(pods), err)
```

Comment:
Not sure that symmetry matters here. Logging an error means you are handling it as per[1] (and I totally agree) and here we are handling errors in two different levels. Also extraneous logs are annoying when debugging.

[1] https://dave.cheney.net/2015/11/05/lets-talk-about-logging